website/docs/source/docs/boxes/base_macos.html.md

```diff
@@ -0,0 +1,172 @@
+---
+page_title: "Creating a macOS Base Box Manually"
+sidebar_current: "boxes-macos-base"
+---
+
+# Creating a macOS Base Box Manually
+
+Prior to reading this page, please check out the [basics of the Vagrant
+box file format](https://www.vagrantup.com/docs/boxes/format.html).
+
+## Pre-requisites
+
+* Download and install [Vagrant](https://developer.hashicorp.com/vagrant/downloads).
+* Install Parallels plugin for Vagrant by executing the command below in Terminal:
+  
+    ```bash
+    $ vagrant plugin install vagrant-parallels
+    ```
+
+* Start Parallels Desktop and create a new macOS virtual machine as outlined  in [KB 125561](http://kb.parallels.com/125561/).
+    <div class="alert alert-info">
+    <p>
+        While creating the VM, give the username as ‘vagrant’ only.
```

Comment:
and if it is a public box, then password = vagrant (recommended according to https://developer.hashicorp.com/vagrant/docs/boxes/base#)

Reply:
Added that mention